wasm-engines/docker/asmble.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:20.04
+
+RUN apt update -y && apt-get install -y wget && \
+    wget https://github.com/cdetrio/asmble/releases/download/0.4.2-fl-bench-times/asmble-0.4.2-fl-bench-times.tar && \
+    tar -xvf asmble-0.4.2-fl-bench-times.tar
+
+# install asmble
+# RUN wget https://github.com/ewasm-benchmarking/asmble/archive/0.4.2-fl-bench-times.tar.gz
```

Comment:
You can upload the release files at https://github.com/ewasm-benchmarking/asmble/releases/edit/0.4.2-fl-bench-times

Anyway, did it myself.